docs/docs-content/getting-started/getting-started.md

```diff
@@ -2,35 +2,95 @@
 sidebar_label: "Getting Started"
 title: "Getting Started"
 description: "Spectro Cloud Getting Started"
-hide_table_of_contents: false
+hide_table_of_contents: true
 sidebar_custom_props:
   icon: "overview"
 tags: ["getting-started"]
 ---
 
-This page gives an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
+This page gives you an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
 full-stack management at scale. Palette's unique capabilities provide end-to-end declarative cluster management, cluster
-monitoring and reconciliation, as well as adherence to the highest security standards. It provides support for public
-cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity of your production
-environments.
+monitoring and reconciliation, as well as enterprise-grade security.
+
+Check out the following short video for an overview of some the key functionality that Palette provides. Spectro Cloud
+Palette provides developers and platform engineers with complete control of their Kubernetes clusters. It provides
+support for public cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity
+or location of your production environments.
+
+<YouTube
+  url="https://www.youtube.com/embed/P9QBOifS_cc"
+  title="Demo | Spectro Cloud Palette - Kubernetes-as-a-Service"
+/>
+
+<br />
+
+## Deploy Kubernetes Clusters with Palette
 
 The first step towards adopting Palette in your organization is to
 [create a login](https://www.spectrocloud.com/get-started). The
 [Palette Free Tier](https://www.spectrocloud.com/free-tier) allows you to experience the benefits of Palette first-hand.
 
-The Getting Started section introduces you to the concepts and workflows you need to begin using Palette. Explore more
-through the following pages.
+We have curated the pages in the Getting Started section to give you a gradual introduction to the fundamental concepts
+and workflows you need to deploy Kubernetes clusters to Palette.
```

Comment:
```suggestion
and workflows you need to deploy and manage Kubernetes clusters through Palette.
```
Changed it to "through Palette" as we don't want the reader to think we are compute platform hosting the k8s cluster :)